.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
+    "env": {
+	"es6": true,
+	"node": true,
+	"jasmine": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+	"sourceType": "module",
+	"ecmaVersion": 6,
+	"ecmaFeatures": {
+	    "jsx": true
+	}
+    },
+    "rules": {
+	"indent": ["error", 2, {"SwitchCase": 1}],
+	"no-else-return": "error",
+	"no-multi-spaces": "error",
+	"no-whitespace-before-property": "error",
+	"camelcase": "error",
+	"new-cap": "error",
+	"no-console": "error",
+	"comma-dangle": "error",
```

Comment:
Set to `always`.

Reply:
always is not an option for all of the rules.